.github/workflows/promptflow-executor-unit-test.yml

```diff
@@ -102,3 +97,17 @@ jobs:
         pythonVersion: 3.9
         coverageThreshold: 0.5
         token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Sending Status
```

Comment:
maybe make this a step template.

On the other hand, given you have just modified the 4 pipelines, would it be better to filter out other pipelines in check_enforcer?

Reply:
Okay. Good  catch.